### ISSUE TITLE
ps is showing detached processes from other apps

### ISSUE THREAD

**Issue by bobzoller** [convox/rack#494]
here's a session demonstrating the issue:

```
$ convox ps --app pick-api
ID            NAME       RELEASE      SIZE  STARTED       COMMAND                                                                                                                                                                                                                                                                                                                                                                                                                                                
d79b1ab26482  scheduler  RMNTVCEVLYC  512   1 hour ago    /usr/bin/profile sh -c 'node deployer.js [snip]'
b6dcba780427  web        RPNNPLIZGZC  256   15 hours ago  /usr/bin/profile sh -c '/start web'                                                                                                                                                                                                                                                                                                                                                                                                                    
f772f1b5bd58  web        RPNNPLIZGZC  256   15 hours ago  /usr/bin/profile sh -c '/start web'

$ convox releases -a ecru | grep RMNTVCEVLYC
RMNTVCEVLYC  2 weeks ago  active

$ convox releases -a pick-api | grep RPNNPLIZGZC
RPNNPLIZGZC  15 hours ago  active

$ convox -v
client: 20160329000338
server: 20160323012659
```

**Comment by nzoschke** [comment]
This must be a bug in the list one-off process path.

**Comment by ddollar** [comment]
This logic has changed quite a lot in the last 5 months so closing this for now. Please reopen if you're still seeing it.

**Comment by jessepollak** [comment]
This is still happening for us. There are two side effects:

1. The process shows up in an entirely unrelated app when we run `convox ps`
2. it's also logging incorrectly (showing the wrong service name -- it should be `web` but it's showing up as `celery`). 

Any other information we can provide?

**Comment by bobzoller** [comment]
still happening for us too, we're on 20160719172302

**Comment by bobzoller** [comment]
this appears to have been fixed for realz in 20160928105531
